src/richie/apps/core/templates/richie/base.html

```diff
@@ -95,7 +95,7 @@
                 </div>
                 <div class="body-footer__brand">
                     <a href="/" class="body-footer__brand__link">
-                        <img src="{% static "richie/images/logo.png" %}" class="body-footer__brand__logo" alt="">
+                        <img src="{% static "richie/images/logo.png" %}" class="body-footer__brand__logo" alt="{{ SITE.name }}">
```

Comment:
add a small test to explain this and secure it?

Reply:
IMO this is pretty basic stuff and there's not tests for the footer AFAIK. Should I start a new test file for the footer to test this?

Reply:
Or add a Django template comment to explain why alt must be set as the site name?
I'm really afraid that these improvements get lost by developpers not aware about accessibility.

Reply:
ok then :man_shrugging: